shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/controler/DatasetApiController.java

```diff
@@ -341,7 +341,9 @@ public void downloadDatasetById(
 		if (dataset.getUpdatedMetadata() != null && dataset.getUpdatedMetadata().getComment() != null) {
 			datasetName += "-" + dataset.getUpdatedMetadata().getComment();
 		}
-
+		if (datasetName.contains("/")) {
```

Comment:
Hi Jean-Come, maybe it would be better to be OS-independent and work with File.SEPARATOR?